src/main/java/io/dashboardhub/pipelinedashboard/listener/AuthenticationListener.java

```diff
@@ -20,5 +21,9 @@ public void onApplicationEvent(final InteractiveAuthenticationSuccessEvent event
         if (user == null) {
             this.userService.save(new User(event.getAuthentication().getName()));
         }
+
+        user = this.userService.findByUsername(event.getAuthentication().getName());
+        user.getLogins().add(new Login());
```

Comment:
Log file will only be looked at when something goes wrong, so not good for Audit purpose.

Could it be incorporated into *Envers*...oh I see using, `lastLoggedIn` field... I LIKE IT!!!